code/parse/parselo.cpp

```diff
@@ -3457,7 +3457,7 @@ int split_str(const char *src, int max_pixel_w, int *n_chars, const char **p_str
 	ignore_until_whitespace = 0;
 
 	// get rid of any leading whitespace
-	while (is_white_space(*src))
+	while (strip_leading_whitespace && is_white_space(*src))
```

Comment:
Probably better just to remove this loop entirely, since it's followed by a similar loop for greyspace stripping